#empty goal
---
#empty result
---
#feedback contains
# the distance between grasping point and tracked object
float64 distance
#the pose of the object
geometry_msgs/Pose object_pose
#the pose of the grasping point
geometry_msgs/Pose grasp_pose
#the arm cartesian velocity at the end effector
float64 cartesian_velocity